src/bibliography/ui/BibliographySelect.tsx

```diff
@@ -45,13 +45,14 @@ export default class BibliographySelect extends Component<
     this.setState({
       selectedOption,
     })
-    this.props.onChange(selectedOption.entry)
+    this.props.onChange(selectedOption)
```

Comment:
Why do you leak implementation details to the event handler? The argument of `onChange` should be the new value (i.e. same type as the `value`), not any data structure specific to the current implementation.

Reply:
i created a new PR dealing with this. [https://github.com/ElectronicBabylonianLiterature/ebl-frontend/pull/41](url)